src/cgtop/cgtop.c

```diff
@@ -672,7 +673,7 @@ static void help(void) {
                "  -n --iterations=N   Run for N iterations before exiting\n"
                "  -b --batch          Run in batch mode, accepting no input\n"
                "     --depth=DEPTH    Maximum traversal depth (default: %u)\n"
-               "  -M --machine=       Show container\n"
+               "  -M --machine=       Show container (incompatible when cgroup path is specified)\n"
```

Comment:
Any reason why you didn't make this new feature a regular option (`-r`, `--root`)? I think that would be a better fit than stealing the extra argv.